### PR TITLE
Fix: reverse index returns wrong response

### LINKED ISSUE #19: write tests with Mocha & Chai to ensure SDK catches leap years
This should be a straightforward and simple task. We can build from what exists in the `/lib/calendar-tests` directory. 

I would like to ensure that both future and past leap years are accounted for, as this is the most vital part we need to make sure is right.

### PR DESCRIPTION
**Pull request by proalgor**
1. Change the access of the current month to take care of  reversed months array `dayjs.months().reverse()` 
2. Also fixes #19

### CODE REVIEW

**Comment by proalgor** [comment]
@yeahmat let me know if any change is needed

**Comment by proalgor** [comment]
@yeahmat Also I think it makes sense to leave the test package.json to use the file package `"@whilethiscompiles/calendar": "file:../calendar"`... cause we are testing the code not what has already been published.

File: lib/calendar/src/index.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const formatDate = ({
   current: string;
   day?: string;
 }): string => {
-  const month = months.indexOf(current) + 1;
+  const month = 12 - months.indexOf(current);
```
Comment (yeahmat) [inline lib/calendar/src/index.ts:49]: Very slick stuff, and great fix 😎

**Review by yeahmat — APPROVED** [review]
Hey @proalgor I really appreciate you taking the time to address these issues. 🥇 That's an incredible fix you have implemented. 

> makes sense to leave the test package.json to use the file package "@whilethiscompiles/calendar": "file:../calendar"... cause we are testing the code not what has already been published.

I agree! I think this will lead into an alternate cleanup ticket to remove the script 

```json
"prod:clean-install:calendar-tests": "cd ./lib/calendar-tests && npm uninstall @whilethiscompiles/calendar && npm install @whilethiscompiles/calendar",
```
from root `package.json` as that command is blindly running each time we update the release. 

---

I'm going to approve these changes and bump the release in the NPM registry to `0.0.5` 

This satisfies the leap year testing and closes #19 ✅ 

Cheers! 🍻

**Pull request merged**